docs/DeveloperGuide.adoc

```diff
@@ -870,24 +870,27 @@ _{More to be added}_
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `member`, unless specified otherwise)
 
 [discrete]
-=== Use case: Delete person
+=== Use case: Delete member
+*System* : AddressBook
+
+*Actor* : Exco member
 
 *MSS*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  Exco member requests to list members.
```

Comment:
Should we use `user` instead of `Exco member`?
Because I think that `Actor` indicates who the possible users are